docs/gatsby-config.js

```diff
@@ -16,29 +16,43 @@ module.exports = {
           1: 'version-1',
         },
         sidebarCategories: {
-          null: ['index', 'getting-started', 'whats-new'],
-          Essentials: [
-            'essentials/schema',
-            'essentials/server',
-            'essentials/data',
+          null: [
+            'index',
+            'getting-started',
+            '[Changelog](https://github.com/apollographql/apollo-server/blob/master/CHANGELOG.md)',
           ],
-          Features: [
-            'features/caching',
-            'features/mocking',
-            'features/errors',
-            'features/data-sources',
-            'features/subscriptions',
-            'features/metrics',
-            'features/graphql-playground',
-            'features/scalars-enums',
-            'features/unions-interfaces',
-            'features/directives',
-            'features/creating-directives',
-            'features/authentication',
-            'features/testing',
-            'features/apq',
-            'features/health-checks',
-            'features/file-uploads',
+          'Defining a Schema': [
+            'defining-a-schema/schema',
```

Comment:
Not opposed, but I'm curious what the rational for using the longer, hyphenated `defining-a-schema` here is?  At a point and time in SEO history, I believe this might have been a great SEO pattern, but I honestly am uninformed enough to know if that's really the case anymore.  

My point is: I tend to err on the side of brevity and would certainly wonder if we shouldn't just have this be reduced to `schema/`?  (i.e. I happen to be a person that types these out a lot from memory, in chat and on forums like Spectrum)